_includes/menu.html

```diff
@@ -2,7 +2,7 @@
 <header class="navbar navbar-default navbar-demo navbar-fixed-top eslint-nav" id="top" role="banner">
   <div class="container">
 
-    <a href="/" class="navbar-brand"><img alt="ESLint" src="/img/logo.svg" itemprop="image">ESLint</a>
+    <a href="{{ site.url }}/" class="navbar-brand"><img alt="ESLint" src="{{ site.url }}/img/logo.svg" itemprop="image">ESLint</a>
```

Comment:
No, because the site would end up at `https://<username>.github.io/eslint.github.io`, so the path `/img/logo.svg` would load the image from `https://<username>.github.io/img/logo.svg`, rather than `https://<username>.github.io/eslint.github.io/img/logo.svg`.

I've also encountered this issue when trying to demo things for this repo.